webapp/src/components/AccordianLayout.tsx

```diff
@@ -0,0 +1,51 @@
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Divider,
+  Typography,
+} from "@mui/material";
+import React from "react";
+import Description from "components/Description";
+
+type Props = {
+  name: string;
+  description: string;
+  docURL: string;
+  children: React.ReactNode;
+};
+
+const AccordianLayout: React.FC<Props> = ({
```

Comment:
Oups! `Accordion` with an `o`.
```suggestion
const AccordionLayout: React.FC<Props> = ({
```